docs/src/routes.js

```diff
@@ -112,7 +112,7 @@ export function getHashUrlByComponentName(name) {
       });
     });
   }
-  return nameToUrlMap[name];
+  return nameToUrlMap[name] || "";
```

Comment:
if the name isn't present in the map, is that a bug? should we throw an exception or return some obvious string like `"<unnamed>"` rather than `""` which might silently fail?

Reply:
this is mainly for the CodeSandbox in the home page, https://codesandbox.io/s/w0z0y5pyy8 
This change will turn `// docs: https://cruise-automation.github.io/webviz/worldview/#undefined` to `// docs: https://cruise-automation.github.io/webviz/worldview/# ` which is the correct URL